src/app/storage/FabricTableImpl.h

```diff
@@ -136,6 +149,9 @@ class FabricTableImpl
     void SetTableSize(uint16_t endpointEntryTableSize, uint16_t maxPerFabric);
     bool IsInitialized() { return (mStorage != nullptr); }
 
+    template <size_t kEntryMaxBytes>
+    CHIP_ERROR IterateEntries(FabricIndex fabric, PersistentStore<kEntryMaxBytes> & store, IterateFnType iterateFn);
```

Comment:
Could use some documentation.  Presumably the callback passed in is then passed the iterator that gets created inside here, and the iterator is going to use `store` to keep its state?  And the iterator is only OK to use until the function returns?